models/yiiModels/EventSearch.php

```diff
@@ -88,7 +86,7 @@ public function attributeLabels() {
      * @return mixed DataProvider of the result or string 
      * \app\models\wsModels\WSConstants::TOKEN if the user needs to log in
      */
-    public function search($sessionToken, $searchParams) {
+    public function searchEvents($sessionToken, $searchParams) {
```

Comment:
Go back to the search name. It is clear that in the EventSearch events will be searched. In all other classes the equivalent method is called search...